zooApplication/src/main/java/com/zoo/zooApplication/resource/BookingResource.java

```diff
@@ -87,4 +86,24 @@ public Response createBooking(@RequestBody CreateBookingRequest bookingRequest,
                         .build();
         return bookingService.findAllBookingByFieldId(searchRequest);
     }
+
+    @ApiOperation(value = "proceed to user information after time slot allocation")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "Resource found")})
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Path("searchByUserInfo")
+    public List<FieldBooking> findByUserInfo
```

Comment:
@Koaha 👍 nice! thanks anh Khoa, có cái này nó hơi subtle nên em quên ko nói lúc làm. This is great rồi, but we still need one more parameter. Ví dụ như hôm đó tới ngày book, thằng đá chạy lên hỏi chủ sân là "hôm bữa em có gọi đặt sân cho hôm nay, blah blah" thì thằng chủ sân nó should be look up result cho bữa đó only thôi nên cái đó require api của mình phải filter by giờ start, not by created time nha, created time có thể dc create từ hồi nảo hồi nao rồi. mà là by start time nha, api thì mình làm ms thôi, để ko rối time zone, epoch la utc time hết nhé. Nên @Koaha add thêm cái parameter này cho cái pr này nhé. Thanks

Reply:
ukie, a mới change lại sort by start time

Reply:
a mới thêm điều kiện truy xuất timeIn từ thời điểm serach trở đi,  (cái này a ko bik trả về là single record hay multi record thì tiện hơn -- mặc dù theo như scenario thì 1 record thoi)